data/json/items/containers.json

```diff
@@ -2791,5 +2801,189 @@
+        "rigid": true,
+        "watertight": true,
+        "max_contains_volume": "3730 ml",
+        "max_contains_weight": "50 kg",
+        "max_item_length": "254 mm"
+      }
+    ],
+    "material": [ "steel", "aluminum" ],
+    "symbol": "□",
+    "color": "light_gray"
+  },
+  {
+    "id": "ammunition_can_50",
+    "type": "GENERIC",
+    "name": "M2A2 ammo can",
+    "copy-from": "ammunition_can_30",
+    "weight": "2360 g",
+    "volume": "8666 ml",
+    "longest_side": "299 mm",
+    "//": "<https://www.cleanammocans.com/new-50-cal-m2a1-usgi-ammo-can.html>",
+    "//2": "External: 29.85 x 15.24 x 19.05 cm, Internal: 27.94 x 13.97 x 17.78 cm ",
+    "price": 1999,
+    "pocket_data": [
+      {
+        "pocket_type": "CONTAINER",
+        "rigid": true,
+        "watertight": true,
+        "max_contains_volume": "6940 ml",
+        "max_contains_weight": "80 kg",
+        "max_item_length": "279 mm"
+      }
+    ]
+  },
+  {
+    "id": "ammunition_can_fat_50",
+    "type": "GENERIC",
+    "name": "PA108 SAW ammo can",
+    "copy-from": "ammunition_can_30",
+    "weight": "2990 g",
+    "volume": "12627 ml",
+    "longest_side": "318 mm",
+    "//": "<https://www.cleanammocans.com/2-pack-new-fat-50-pa108-saw-ammo-cans.html>",
+    "//2": "External: 31.75 x 18.42 x 21.59 cm, Internal: 29.85 x 17.15 x 20.96 cm ",
+    "price": 1999,
+    "pocket_data": [
+      {
+        "pocket_type": "CONTAINER",
+        "rigid": true,
+        "watertight": true,
+        "max_contains_volume": "10730 ml",
+        "max_contains_weight": "120 kg",
+        "max_item_length": "298 mm"
+      }
+    ]
+  },
+  {
+    "id": "ammunition_can_pa120",
+    "type": "GENERIC",
+    "name": "PA-120 ammo can",
+    "copy-from": "ammunition_can_30",
+    "description": "Huge metal box with handles on each end.  Pretty heavy.",
+    "weight": "6350 g",
+    "volume": "18572 ml",
+    "longest_side": "470 mm",
+    "//": "<https://www.cleanammocans.com/2-pack-40mm-ammo-cans.html>",
+    "//2": "External: 46.99 x 15.56 x 25.40 cm, Internal: 43.82 x 13.97 x 24.13 cm",
+    "price": 2500,
+    "pocket_data": [
+      {
+        "pocket_type": "CONTAINER",
+        "rigid": true,
+        "watertight": true,
+        "max_contains_volume": "14772 ml",
+        "max_contains_weight": "160 kg",
+        "max_item_length": "438 mm"
+      }
+    ]
+  },
+  {
+    "id": "ammunition_box_grenade",
+    "type": "GENERIC",
+    "category": "container",
+    "name": "M289 container",
+    "description": "Small and thick cylindrical container, designed to store 1 grenade.",
+    "weight": "411 g",
+    "volume": "590 ml",
+    "pocket_data": [
+      {
+        "pocket_type": "CONTAINER",
+        "rigid": true,
+        "watertight": true,
+        "max_contains_volume": "550 ml",
+        "max_contains_weight": "5000 g"
+      }
+    ],
+    "material": [ "steel" ],
+    "symbol": "0",
+    "color": "black"
+  },
+  {
+    "id": "ammunition_box_308",
```

Comment:
I don't think we need this many individually defined ammo boxes, esp. when we have need of a large number of potential ammo boxes that should be defined given how many calibers are represented ingame that should spawn inside them(down with dropping a large pile of loose cartridges!).

Reply:
Not sure i understand what you mean.

Reply:
I think this is a reasonable solution for military. 

In the future paper boxes should also be a thing though but it's a bunch of work to get an automated solution working.

Reply:
but what about the amount of rounds? the box for 20 rounds is not the same as box for 50 rounds, and 20 223 round box is not the same as 20 308 round box. what number we need to pick for rounds to not make 5+ different boxes for each round (which is a huge ton of a boxes that no one want to make, use, and hanlde, obviously)
i use 100x9mm, 30x223, 20x300, 20x12 and 10x50 round boxes for my purpose, but i came to this numbers using ammo cans standarts, so they can be unperfect for some purposes, i think

Reply:
what I'm saying is what you are doing and have added is fine for now IMO.